src/main/asciidoc/dataobjects.adoc

```diff
@@ -673,6 +673,16 @@ Set the maximum HTTP chunk size
 +++
 Set the maximum pool size for connections
 +++
+|[[maxStreams]]`maxStreams`|`Number (int)`|
++++
+Set the maximum of concurrent concurrency for an HTTP/2 connection, this limits the number
+ of streams the client will create for a connection. The effective number of stream for a
+ connection can be lower than this value when the server has limited this value lower than
+ this value.
```

Comment:
"when the server has limited this value lower than this value." could be changed to "when the server has a lower limit." (better English)